aiida/plugins/entry_point.py

```diff
@@ -30,11 +30,18 @@
 ENTRY_POINT_STRING_SEPARATOR = ':'
 
 
-@functools.lru_cache(maxsize=1)
+@functools.cache
 def eps():
     return _eps()
 
 
+@functools.lru_cache(maxsize=100)
```

Comment:
interesting, so do I understand correctly that `eps()` is cached, nothing is being read from disk here, and processing the few entry points in memory is still taking 25ms?

Can you please document this function to explain the quadruple loop problem here, and why this additional cache is necessary?

Reply:
Yes, I'll write a comment. You can look at the cProfile info on the forum. https://aiida.discourse.group/t/why-is-aiidalab-base-widget-import-so-slow/32/16?u=danielhollas

Together with the fact that we were calling this during each Node class build (in a metaclass), this completely explained why the `aiida.orm` import was so slow. See #6091